event/views.py

```diff
@@ -42,7 +43,26 @@ def create_event(request, user_id):
 def view_event(request, user_id):
     event_id = request.GET["id"]
     event = Event.manager.get(id=event_id)
-    return render(request, 'event/event_info.html', {'event': event})
+    if request.method == "POST":
+        Teams.generate_teams(event_id)
+
+    team1 = []
+    team2 = []
+    users_events = UserEvent.objects.filter(eventID=event_id)
+    if len(users_events) > 0:
+        if users_events[0].teamID:
+            teams = list(set([user_event.teamID
+                              for user_event in users_events]))
+            team1 = [user.userID.user.username
+                     for user in UserEvent.objects.filter(teamID=teams[0].id)]
+            team2 = [user.userID.user.username
+                     for user in UserEvent.objects.filter(teamID=teams[1].id)]
```

Comment:
Combining the two conditions together allows you to go back one indentation
```suggestion
    if len(users_events) > 0 and users_events[0].teamID:
        teams = list(set([user_event.teamID for user_event in users_events]))
        team1 = [user.userID.user.username for user in UserEvent.objects.filter(teamID=teams[0].id)]
        team2 = [user.userID.user.username for user in UserEvent.objects.filter(teamID=teams[1].id)]
```

Reply:
This implementation is structured in this particular way due to the scenario where the length of the `user_event` is 0. In such cases, attempting to access `user_event[0].teamID` would result in an error.